src/components/shared/CoinIcons/DynamicCoin.style.tsx

```diff
@@ -0,0 +1,38 @@
+import styled, { StyledComponent } from 'styled-components'
+import { palette } from 'styled-theme';
+
+type SizeType = { [key: string]: string}
+const sizes: SizeType = {
+  big: '58px',
+  normal: '42px',
+  small: '32px',
+};
+
+type FontSizeType = { [key: string]: string}
+const fontSizes:FontSizeType = {
+  big: '16px',
+  normal: '12px',
+  small: '9px',
+};
+
+type Props = {
+  type: string,
+  size: string,
+}
+
+export const DynamicCoinWrapper: StyledComponent<'div',any,{},never> = styled.div<Props>`
```

Comment:
```
asgardex-electron/src/components/shared/CoinIcons/DynamicCoin.style.tsx
  23:57  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
```
To avoid any type or lint issues (Unexpected `any`), just remove this type definition of `StyledComponent ...`. It will be inferred by `tsc` for you.

```
export const DynamicCoinWrapper = styled.div<Props>`
...
`
```